cli/smartcontract/smart_contract.go

```diff
@@ -443,6 +464,16 @@ func invokeInternal(ctx *cli.Context, withMethod bool, signAndPush bool) error {
 		}
 	}
 
+	if len(args) >= hashesForVerifyingStart && hashesForVerifyingStart > 0 {
+		for i, c := range args[hashesForVerifyingStart:] {
+			h, err := parseUint160(c)
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("failed to parse hash for veryfying #%d: %v", i+1, err), 1)
```

Comment:
Let's merge #1281 first and then use '%w' here instead of '%v'.

Reply:
Oh, it's 2.0, so doesn't really matter.

Reply:
Fixed this.